app/models/api_request.rb

```diff
@@ -1,10 +1,20 @@
 class ApiRequest
+  PAGE_LIMIT = 25
+
   def response
     @response ||= get_response
   end
 
   private
 
+  def default_params
```

Comment:
[suggestion] I think it will be unlikely that all the systems will agree enough on the params for this to be useful. At best, it's going to result in unnecessary parameters being sent. At worst it'll result in weird bugs. Even within a system you won't need paging params for show routes